### DISCUSSION TITLE
[BUG] Avatar missing on page 2

### DISCUSSION THREAD

**Discussion by stevolucidity** [HEIGE-PCloud/DoIt#386] [Q&A] [answered]
### Describe the bug 描述你遇到的错误

On page 2 of the blog, the avatar is not found. 
Failed to load resource: the server responded with a status of 404 () https://blog.simons.earth/page/2/avatar.png

### Expected behavior 期待的行为

I expect to see the avatar at the top of the page.

### Screenshots 屏幕截图

Please see attached.

### Build Environment 构建环境

- macOS Monterey Version 12.0.1
- Theme version latest (https://github.com/HEIGE-PCloud/DoIt/commit/f7386ced624855c397b9d2c85b554335c0e79cb9)
- Hugo version hugo v0.88.1+extended darwin/amd64

Page 1:
![Image 10-12-2021 at 21 21](https://user-images.githubusercontent.com/44602820/145642932-2b621ae8-fb57-4e47-846e-2c58c9a645f0.jpg)

Page 2:
![Image 10-12-2021 at 21 09](https://user-images.githubusercontent.com/44602820/145642674-142b6030-6e7e-4e3d-994b-a344064273ef.jpg)

Browser message:
![Image 10-12-2021 at 21 10](https://user-images.githubusercontent.com/44602820/145642692-87628038-1416-4f1f-9e9c-18bf85bd6f87.jpg)

I'm new. Apologies if I'm doing something wrong.

**Comment by HEIGE-PCloud** [comment, marked as answer]
An absolute path is needed.

https://github.com/evoLucidity/simonsblog.github.io/blob/57256b132a51509b38403152cf432681e72768d1/config.toml#L137

Change `avatar.png` to `/avatar.png` will fix it.

**Comment by stevolucidity** [comment]
Perfect, thanks! 😊
